qiskit/extensions/standard/rxx.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Two-qubit XX-rotation gate.
+"""
+import numpy as np
+from qiskit.circuit import Gate
+from qiskit.circuit import QuantumCircuit
+
+
+class RXXGate(Gate):
+    """Two-qubit XX-rotation gate.
+
+    This gate corresponds to the rotation RZZGate(θ) = exp(-1j * θ * X⊗X)
+    """
+
+    def __init__(self, theta):
+        """Create new rxx gate."""
+        super().__init__("rxx", 2, [theta])
+
+    def _define(self):
+        """Calculate a subcircuit that implements this unitary."""
+        from qiskit.quantum_info.synthesis.two_qubit_decompose import two_qubit_cnot_decompose
+        self.definition = two_qubit_cnot_decompose(self.to_matrix())
```

Comment:
I think `self.definition` should be a list.

Reply:
True, but I'm not sure what the decomposition for this one is...

Reply:
Can you use the decomposition from #3114 ?